e2e/rsmgp_example_test/test_simple/test.yml

```diff
@@ -0,0 +1,27 @@
+query: >
+    CALL libmemgraph_query_module_in_rust_example.test_procedure() YIELD *;
+
+output:
+    - first_label: ""
+      has_L3_label: false
+      labels_count: 0
+      name_property: "unknown"
+      properties_string: "age: 29"
```

Comment:
@jmatak is it possible to test the whole vertex here?

Reply:
Currently, it is not supported, however, if there is a need for that it can be easily extended with the new pymgclient and gqlalchemy modules.

Reply:
It's not critical, but it would be nice to have :smile: